JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Vue/App.vue

```diff
@@ -17,7 +16,7 @@
       <DxPaging :enabled="false"/>
       <DxSelection mode="multiple"/>
       <DxColumn
-        :width="55"
+        width="55"
```

Comment:
Why?

Reply:
thought that it's best practise not to bind consts, [but it's not](https://ru.vuejs.org/v2/guide/components-props.html#%D0%9F%D0%B5%D1%80%D0%B5%D0%B4%D0%B0%D1%87%D0%B0-%D1%87%D0%B8%D1%81%D0%B5%D0%BB)